metriq_gym/dispatch_qv.py

```diff
@@ -16,24 +16,30 @@
 def main():
     args = parse_arguments()
 
+    try:
+        params = load_and_validate(args.input_file)
+    except ValidationError as e:
+        logging.error(f"JSON validation error: {e.message}")
+        return 1
+
     logging.info(
-        f"Dispatching Quantum Volume job with n={args.num_qubits}, shots={args.shots}, trials={args.trials}, backend={args.backend}, confidence_level={args.confidence_level}, jobs_file={args.jobs_file}"
+        f"Dispatching {params['benchmark_name']} job with n={params["num_qubits"]}, shots={params["shots"]}, trials={params["trials"]}, backend={args.backend}, confidence_level={params["confidence_level"]}, jobs_file={args.jobs_file}"
     )
 
     job_manager = JobManager(args.jobs_file)
 
     result = dispatch_bench_job(
-        args.num_qubits, args.backend, args.shots, args.trials, args.provider
+        params["num_qubits"], args.backend, params["shots"], params["trials"], args.provider
```

Comment:
[question]: Should we be providing things like `provider` and `backend` also in the schema? It might be nice to have all of the configuration contained in one spot for given run for the sake of posterity and replication of results, etc.

Reply:
My rationale for not including those is that one may want to run the exact same benchmark on different provider/backend. I am envisioning the following command:
```
python dispatch.py qv_job.json --provider ibmq --backend ibm-sherbrooke
python dispatch.py qv_job.json --provider quantinuum --backend h1
```
Where qv_job.json doesn't change.
However, in the result file, we would still save parameters coming from the cli args.
Up for discussion though.

Reply:
Yeah, I'm not sure what the "right" way is to do this, but let's go with your assumption here and if we need to change it, we can.

Reply:
Exactly, we can always change it later.